recipes/xeofs/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "xeofs" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xeofs-{{ version }}.tar.gz
+  sha256: ab3700370349dba7b4bda261471c1af01f9f00ed5bd4791e74098a5bb4537743
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
```

```suggestion
    - python >=3.8
```

I know this is listed upstream but please remove the `<4.0`part. When it comes to that point conda-forge will most likely have a bot that will manage this and we don't want to restrict ourselves with that yet.